tools/toolchains/__init__.py

```diff
@@ -191,7 +191,7 @@ def get_symbols(self, for_asm=False):
                 labels = self.get_labels()
                 self.cxx_symbols = ["TARGET_%s" % t for t in labels['TARGET']]
                 self.cxx_symbols.extend(["TOOLCHAIN_%s" % t for t in labels['TOOLCHAIN']])
-
+                
```

Comment:
this empty space should be removed (how come astyle did not catch that)

Reply:
@0xc0170 Congrats on the new test case!

Reply:
Fixed now.